include/libtorrent/session_handle.hpp

```diff
@@ -456,6 +456,14 @@ namespace libtorrent {
 		// posted, regardless of the alert mask.
 		void dht_live_nodes(sha1_hash const& nid);
 
+		// Query the DHT to retrieve a sample of the infohashes that the specific
+		// node currently have in their storage.
```

Comment:
"the specific node", I take it this should say something like: "the node specified by ``ep``". Is that right, or am I misunderstanding this call?